tutorials/euclid_access/4_Euclid_intro_PHZ_catalog.md

```diff
@@ -172,10 +171,10 @@ The _unif fluxes are: "Unified flux recomputed after correction from galactic ex
 We specify the following conditions on our search:
 - We select just the galaxies where the flux is greater than zero, to ensure the appear in all four of the Euclid MER images.
 - Select only objects in a circle (search radius selected below) around our selected RA and Dec
-- phz_classification =2 means we select only galaxies
-- Using the phz_90_int1 and phz_90_int2, we select just the galaxies where the error on the photometric redshift is less than 20%
+- `phz_classification = 2` means we select only galaxies
+- Using the `phz_90_int1` and `phz_90_int2`, we select just the galaxies where the error on the photometric redshift is less than 20%
 - Select just the galaxies between a median redshift of 1.4 and 1.6
-- We search just a 3 arcminute box around an RA and Dec
+- We search just a 5 arcminute box around an RA and Dec
```

Comment:
Note: I made the narrative consistent with the code. We could choose to go the other way around though (and use the 3'x3' in the code)